examples/c/5_series/ahrs/README.md

```diff
@@ -0,0 +1,187 @@
+# 5 Series AHRS Example (C)
+
+This example demonstrates how to configure and use a MicroStrain 5-series AHRS device with the MIP SDK using the C API.
+
+## Overview
+
+The example showcases the basic setup and operation of a 5-series AHRS device, including:
+- Device initialization and communication
+- Filter message format configuration
+- Gyro bias capture
+- Filter initialization and heading source configuration
+- Real-time data streaming and display
+
+## Configuration
+
+The example uses the following default settings:
```

Comment:
In general for this and all the example READMEs, it would be nice to note that these variables exist at the top of the file and can be changed. You mention it down below, but to someone inexperienced, it might not be super obvious